io.mjml.mjml-app.yaml

```diff
@@ -0,0 +1,74 @@
+app-id: io.mjml.mjml-app
```

Comment:
Can you remove this file so only the source code one is being build?

Reply:
done.